libbpf-tools/memleak.c

```diff
@@ -870,6 +875,10 @@ int print_outstanding_allocs(int allocs_fd, int stack_traces_fd)
 
 		// filter invalid stacks
 		if (alloc_info.stack_id < 0) {
+			/* handle stack id errors */
+			nr_missing_stacks++;
+			has_collision = CHECK_STACK_COLLISION(alloc_info.stack_id);
```

Comment:
-EFAULT or -ENOMEM could also be returned. So let us just do nr_missing_stacks and nr_collision_stacks so people know whether other kinds of errors may happen (although in general it should not).

Reply:
If any additional changes are needed, please let me know.
Thank you.